json/src/test/kotlin/ktx/json/jsonSerializerTest.kt

```diff
@@ -0,0 +1,84 @@
+package ktx.json
+
+import com.badlogic.gdx.utils.Json
+import com.badlogic.gdx.utils.JsonValue
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import io.kotlintest.matchers.shouldEqual
+import io.kotlintest.matchers.shouldThrow
+import org.junit.Test
+
+@Suppress("unused")
+class `should implement ReadOnlyJsonSerializer with no 'write' method overriden`<T> : ReadOnlyJsonSerializer<T> {
+  // Guarantees that [write] method is optional to implement.
+
+  override fun read(json: Json, jsonData: JsonValue, type: Class<*>?): T {
+    throw NotImplementedError()
+  }
+
+  @Test
+  fun `read throws NotImplementedError`() {
```

Comment:
`write`, not read. I wanted you to test the default implementation of `write` method that comes from the `ReadOnlyJsonSerializer,` not the `read` method that you implemented here.

Reply:
I can see that you've added a test for that separately, so you can remove this test completely.

Reply:
I'm quite inattentive today. Sorry. I can remove it, no problem.

Reply:
It's really OK, might have been my bad since I included the comment next to the class. Excessive tests are better than missing tests. ;)

We generally tend to test only the actual libraries and not the utility classes used only in tests, hence the comment.